src/renderer/components/reader/Reader.tsx

```diff
@@ -462,7 +455,7 @@ export class Reader extends React.Component<ReaderProps, ReaderState> {
             // https://github.com/electron/electron/blob/v3.0.0/docs/api/breaking-changes.md#webframe
             // publicationJsonUrl is READIUM2_ELECTRON_HTTP_PROTOCOL (see convertCustomSchemeToHttpUrl)
             // publicationJsonUrl_ is https://127.0.0.1:PORT
-            response = await fetch(publicationJsonUrl_);
+            response = await fetch(publicationJsonUrl);
```

Comment:
Can we please rename `publicationJsonUrl` as `queryParams.pub`, and `publicationJsonUrl_` as `publicationJsonUrl` in the comment immediately above this line of code, thanks :)

Reply:
There is a regression in this PR, because `installNavigatorDOM()` is still called with `publicationJsonUrl`, when it should have been adjusted to the `READIUM2_ELECTRON_HTTP_PROTOCOL` `queryParams.pub` URL. I am working on a fix...

Reply:
Fixed via https://github.com/readium/readium-desktop/commit/36fbca41c4307b4a0703164cd9873bc7364d47ea